package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "unity-cache",
-  "version": "2.1.0",
-  "description": "Cache abstraction around localforage.",
+  "version": "2.2.0",
```

Comment:
You updated node to major version. I think it should been updated to major.

Reply:
Node is used only in testing environment. I'm not sure about the major version.